src/applications/appeals/shared/constants.js

```diff
@@ -70,13 +74,16 @@ export const MAX_FILE_SIZE_BYTES = MAX_FILE_SIZE_MB * 1024 ** 2; // binary based
  **** MAX LENGTH ****
  */
 
-// Values from Lighthouse maintained schema v2
-// see https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/config/schemas/v2/200996.json
+// Values from Lighthouse maintained schema v1 & v2
+// v1 - see https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/config/schemas/v1/10182.json
+// v2 - see https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/config/schemas/v2/200996.json
 export const MAX_LENGTH = {
-  SELECTIONS: 100, // submitted issues (not in schema)
+  // used in 995 and 996
   ISSUE_NAME: 140,
-  DISAGREEMENT_REASON: 90,
   EMAIL: 255,
+  // used in 995 and 10182
+  SELECTIONS: 100, // submitted issues (not in schema)
+  // used in all 3 forms
   COUNTRY_CODE: 3,
   AREA_CODE: 4,
```

Comment:
SC has this value named as `PHONE_COUNTRY_CODE` and `PHONE_AREA_CODE`. Please rename these here and in the HLR & NOD code.